packages/web-components/src/components/cbp-structured-list/cbp-structured-list.stories.tsx

```diff
@@ -251,13 +252,68 @@ const StructuredListWithGridTemplate = ({ striped, selectable, showHeader, heade
+      `;
+};
+export const StructuredListCollection = StructuredListCollectionTemplate.bind({});
+StructuredListCollection.argTypes = {
+  listItems: {
+    description: 'Configure various aspects of the list items within the structured list.',
+    control: 'object',
+  },
+}
+StructuredListCollection.args = {
+  listItems: [
+    {
+      content: "<div style='display:flex;flex-direction: column; gap: 0.5rem;'><span><cbp-icon name='arrow-right' color='var(--cbp-link-color)' size='1rem'></cbp-icon><cbp-link href='#' target='_self' style='margin-inline-start: 0.5rem'>Internal Link</cbp-link></span><span>Description text</span><span> <cbp-icon name='user' size='1rem'></cbp-icon> <i>https://www.text-link.com/help-me</i></span><cbp-tag class='hydrated' style='width: 50px'> Tag </cbp-tag></div>",
+      color: 'default',
+      selected: false
+    },
+    {
+      content: "<div style='display:flex;flex-direction: column; gap: 0.5rem;'><span><cbp-icon name='arrow-right' color='var(--cbp-link-color)' size='1rem'></cbp-icon><cbp-link href='#' target='_self' style='margin-inline-start: 0.5rem'>Internal Link</cbp-link></span><span>Description text</span><span> <cbp-icon name='user' size='1rem'></cbp-icon> <i>https://www.text-link.com/help-me</i></span><cbp-tag class='hydrated' style='width: 50px'> Tag </cbp-tag></div>",
+      color: 'default',
+      selected: false
+    },
+    {
+      content: "<div style='display:flex;flex-direction: column; gap: 0.5rem;'><span><cbp-icon name='arrow-right' color='var(--cbp-link-color)' size='1rem'></cbp-icon><cbp-link href='#' target='_self' style='margin-inline-start: 0.5rem'>Internal Link</cbp-link></span><span>Description text</span><span> <cbp-icon name='user' size='1rem'></cbp-icon> <i>https://www.text-link.com/help-me</i></span><cbp-tag class='hydrated' style='width: 50px'> Tag </cbp-tag></div>",
+      color: 'default',
+      selected: false
+    },
+    {
+      content: "<div style='display:flex;flex-direction: column; gap: 0.5rem;'><span><cbp-icon name='arrow-right' color='var(--cbp-link-color)' size='1rem'></cbp-icon><cbp-link href='#' target='_self' style='margin-inline-start: 0.5rem'>Internal Link</cbp-link></span><span>Description text</span><span> <cbp-icon name='user' size='1rem'></cbp-icon> <i>https://www.text-link.com/help-me</i></span><cbp-tag class='hydrated' style='width: 50px'> Tag </cbp-tag></div>",
+      color: 'default',
+      selected: false
+    },
+    {
+      content: "<div style='display:flex;flex-direction: column; gap: 0.5rem;'><span><cbp-icon name='arrow-right' color='var(--cbp-link-color)' size='1rem'></cbp-icon><cbp-link href='#' target='_self' style='margin-inline-start: 0.5rem'>Internal Link</cbp-link></span><span>Description text</span><span> <cbp-icon name='user' size='1rem'></cbp-icon> <i>https://www.text-link.com/help-me</i></span><cbp-tag class='hydrated' style='width: 50px'> Tag </cbp-tag></div>",
+      color: 'default',
+      selected: false
+    },
+  ]
+}
+
```

Comment:
You also need to set the `showHeader` arg to false for this story because it's set to true at the high level.